azure-pipelines.yml

```diff
@@ -0,0 +1,62 @@
+trigger:
+  branches:
+    include:
+    - main
+    - develop
+pr:
+  branches:
+    include:
+    - '*' 
+
+pool:
+  vmImage: 'windows-latest'
+
+steps:
+- checkout: self
+  clean: true
+
+- task: UseDotNet@2
+  displayName: Set .NET SDK to 5.x
+  inputs:
+    packageType: 'sdk'
+    version: '5.x'
+
+- task: SonarCloudPrepare@1
+  displayName: Prepare Sonarcloud
+  inputs:
+    SonarCloud: 'Sonarcloud'
+    organization: 'dept'
+    scannerMode: 'MSBuild'
+    projectKey: 'dtnl-livelight-website'
+    projectName: 'dtnl-livelight-website'
```

Comment:
These should probably come from pipeline variables.

Reply:
Thanks for the suggestion. The hardcoded values has been replaced with variables.